tests/Integration/Behaviour/Features/Scenario/Store/store.feature

```diff
@@ -10,3 +10,26 @@ Feature: store
       Then the store "store1" should have status disabled
       When I toggle "store1"
       Then the store "store1" should have status enabled
+      
+  Scenario: Enabling and disabling multiple stores in bulk action
+    When I add new store "StorePau" with following properties:
+      | name      | StorePau               |
+      | enabled   | true                   |
+      | address1  | 1 rue de la republique |
+      | city      | Pau                    |
+      | latitude  | 43.2951                |
+      | longitude | -0.370797              |
+      | country   | France                 |
+    And I add new store "StoreSerresCastet" with following properties:
+      | name      | StoreSerresCastet      |
+      | enabled   | true                   |
+      | address1  | 1 rue de la foire      |
+      | city      | Serres-Castet          |
+      | latitude  | 43.2951                |
+      | longitude | -0.370797              |
+      | country   | France                 |
+    Then stores "StorePau, StoreSerresCastet" should be enabled
+    When I disable multiple stores "StorePau, StoreSerresCastet" using bulk action
+    Then stores "StorePau, StoreSerresCastet" should be disabled
+    When I enable multiple stores "StorePau, StoreSerresCastet" using bulk action
+    Then stores "StorePau, StoreSerresCastet" should be enabled
```

Comment:
May I suggest adding 3rd store excluded from bulk actions ? just to make sure this store doesn't get toggled by a bug later.

Reply:
Another shop added for test and test done to check that the status doesn't change